include/openmc/particle.h

```diff
@@ -181,6 +181,15 @@ class Particle {
     int delayed_group; //!< particle delayed group
   };
 
+  //! Saved ("banked") state of primary source particle, for particle restart
+  struct RestartBank {
+	    Position r;
+	    Direction u;
+	    double E;
+	    double wgt;
```

Comment:
The OpenMC Style Guide for C++ says to use two spaces per indentation level. The other struct definitions in this file have conformed to this guide, so it's probably best if this one does too.